maquetado/scss/details.scss

```diff
@@ -18,15 +18,34 @@
   margin-left: $element-text-margin-left;
 }
 
-.book-info {
-  flex-direction: column;
-  width: $book-info-width;
+.book-image {
+  width: 100%;
+  height: 100%;
 }
 
-.book-image {
+.book-image-container {
+  position: relative;
   height: $book-image-height;
   width: $book-image-width;
   margin: $book-image-margin;
+
+  &:after {
+    transform: rotate(20deg);
+    background: url('images/badge.png') no-repeat;
+    background-size: contain;
+    content: "";
+    position: absolute;
+    height: $badge-height;
+    width: $badge-width;
+    right: $badge-position-right;
+    top: $badge-position-top;
```

Comment:
Same as the other PR. In addition to the comments I left there we also have that this specific kind of abstraction makes it harder to follow, because we are talking about values that are only used once.